src/ui/components/Library/Invitations.tsx

```diff
@@ -9,7 +9,7 @@ export default function Invitations({}) {
     return null;
   }
 
-  const displayedWorkspaces = [...pendingWorkspaces!].sort((a, b) =>
+  const displayedWorkspaces = [...(pendingWorkspaces || [])].sort((a, b) =>
```

Comment:
these small fixes are great, but should be in their own PRs that land quickly. Like i think this broke for me recently in dev

Reply:
Yep, you're right.

Reply:
https://github.com/RecordReplay/devtools/pull/4453/files